modules/download-images-using-offline-bundle.adoc

```diff
@@ -9,6 +9,8 @@ You can download {product-title} images by using an offline bundle.
 
 .Procedure
 
+// TODO !!!
+
 . Download the following image bundles:
 * Images for all {product-title} components (except Collector) and the roxctl CLI: link:https://install.stackrox.io/{rhacs-version}/image-bundle.tgz[https://install.stackrox.io/{rhacs-version}/image-bundle.tgz]
 * {product-title} Collector image: link:https://install.stackrox.io/{collector-version}/image-collector-bundle.tgz[https://install.stackrox.io/{collector-version}/image-collector-bundle.tgz]
```

Comment:
Hmm, since offline bundles are hosted on stackrox.io, I'm not sure they are still used by many customers. ~Maybe this doesn't need to be touched.~ Nvm, saw that it's about the collector version.

Reply:
@msugakov I think I asked you to sign up at `signup.stackrox.io` or `help.stackrox.com` when you started. Those credentials are what can be used for `install.stackrox.io`

Reply:
Okay, the publishing code is in CircleCI `config.yml`, l.4560ff. However, `collector-version` doesn't even seem to be correct here, so I think just changing this to `rhacs-version` is fine. What I'm worried about is that the offline bundle `.img` files actually include the tag, and that tag might be the old one (`scripts/offline-bundle/create.sh`). Imho this is nothing to hold the release for, as I believe they are virtually unused.

Reply:
Collector bundles indeed go with `rhacs-version` in the bucket.
![image](https://user-images.githubusercontent.com/537715/151542312-851f222a-54e8-4a07-ad85-016482d2ff02.png)

I'll look to fix `scripts/offline-bundle/create.sh`.

Reply:
Thanks @misberner , this indeed worked.

> Those credentials are what can be used for install.stackrox.io

Reply:
https://github.com/stackrox/stackrox/pull/472